company/welcome_pack.md

```diff
@@ -66,6 +66,7 @@ Note that Google office applications (Docs, Sheets) are the preferred format for
 * [ ] LastPass (with 2FA)
 * [ ] Trello (with 2FA unless using Google Auth to login to Trello)
 * [ ] Continuous Feedback
+* [ ] Setup an account with [Gravatar](https://en.gravatar.com/) - You can use this to add a shared profile picture for multiple services.
```

Comment:
We should add something about avatars needing your face, ideally from a professional picture.

Reply:
Can you suggest wording? ` You should add a profile picture that is clearly you, ideally professional looking portrait.`?

Reply:
Also, should we explicitly state they should be "G" rated?

Reply:
Should a photo policy be in a separate place, in a different PR? we already have a requirement elsewhere to have a photo for the website

Reply:
Seems like rules you add after they're broken to me, I'd reasonably expect any new starter to know what would be suitable